token_review.go

```diff
@@ -174,13 +174,14 @@ func parseResponse(resp *http.Response) (*authv1.TokenReview, error) {
 
 // mock review is used while testing
 type mockTokenReview struct {
-	saName      string
-	saNamespace string
-	saUID       string
+	saName            string
+	saNamespace       string
+	saUID             string
+	checkPooledClient bool
```

Comment:
What's the status of this PR? Looks like this is currently unused, and I'm not sure that you'll need it, unless it causes you difficulties while writing the test.

Reply:
Yes ideally I wanted to check or have some indication we are using the `PooledClient` I think it does cause a lot of complications as the only way of getting access to that information is by looking at the `Transport`. Do you have any other suggestions on how to make sure the connection is being reused? The only option I can think of is using client trace to check whether the request's underlying tcp connection was re-used. Example: https://golang.cafe/blog/golang-httptrace-example.html

Reply:
I've left a comment on the mock function code in response to this.